pub-quiz.py

```diff
@@ -24,7 +36,10 @@
         print(option)
     
     # Get the user's answer
-    user_answer = input("Your answer (A, B, C, D): ").strip().upper() # Ensuring the input is uppercase for comparison
+    if question == "The capital of England is London. True or False?":
```

Comment:
I believe it would be beneficial to use more common attribute check for deciding to adjust the answer handle? 
Like if options == ["True", "False"]
In this case you could use many questions with answer options true/false and dont need to write another handler (if clause)